RELEASE.txt

```diff
@@ -32,29 +32,55 @@ the corresponding release branch.
      the affected functions, elaborating on the changes and their
      consequences. If possible, organize semantically close PRs in groups.
 
-  5. Rename the file to ``doc/release/release_<major>.<minor>.txt``
+  5. Rename the file to ``doc/release/release_<major>.<minor>.rst``
 
-  6. Copy ``doc/release/release_template.txt`` to
-     ``doc/release/release_dev.txt`` for the next release.
+  6. Copy ``doc/release/release_template.rst`` to
+     ``doc/release/release_dev.rst`` for the next release.
 
-  7. Copy relevant deprecations from ``release_<major>_<minor>.txt``
+  7. Copy relevant deprecations from ``release_<major>.<minor>.rst``
      to ``release_dev.txt``.
 
 - Submit the release notes for review by other project maintainers:
 
   - Create a PR from v<major>.<minor>.x branch to master (at this point,
     the difference should show the full contents of the release notes).
   - Discuss with others, and make the changes directly to v<major>.<minor>.x branch.
-  - Once the consensus is found, ask the project maintainers to merge the PR.
+  - Once the consensus is found, ask the project maintainers to merge
+    the PR.
+  - Cherry pick the change onto the release branch.
 
-- In the master branch, update the version number in ``skimage/__init__.py``
+- On the master branch, update the version number in ``skimage/__init__.py``
   to the next ``.dev0`` version, commit, and push. This should follow PEP440
   meaning that the appropriate version number would look something like
   ``0.20.dev0`` with the period between ``20`` and ``dev`` and a trailing
-  ``0`` immediately after ``dev``. 
+  ``0`` immediately after ``dev``.
 
-- Back on the release branch, update the version number to stable in
-  ``skimage/__init__.py``, commit, and push.
+- On the master branch, edit ``doc/source/_static/docversions.js``,
+  add the release, e.g., `0.15.x`, and commit.  Cherry-pick this
+  change back to the release branch.
+
+- Back on the release branch, update the version number to match the release
+  version in ``skimage/__init__.py``, commit, and push.
+
+- Update the release docs:
+
+  - On the release branch, build a clean version of the docs. In the
+    root directory, run ``pip install .``.
+  - In the ``doc/`` directory:
+    - Build using
+      ``make clean; make html; make gh-pages``.
+    - In the ``gh-pages/`` directory:
+      - Update the symlink to ``stable`` and commit.
+      - Upload the docs: ``git push origin gh-pages``.
+
+At this point, you are ready to tag the new version.  This "seals the
+deal", so we want to make sure everything is working.  Go to
+https://travis-ci.org/scikit-image/scikit-image/branches, click on the
+release branch, and ensure that all checks have passed for the latest
+build.  Do the same for AppVeyor at
```

Comment:
@stefanv this is necessary, but not sufficient to check that the build will work on `scikit-image-wheels`. Is there a way to check *those* builds on the latest release tip *without* making a release?